src/payment/authorize-net-direct-post.md

```diff
@@ -1,11 +1,14 @@
 ---
-title: Authorize.Net Direct Post (Deprecated)
+title: Authorize.Net Direct Post - Deprecated
 ---
 
-Authorize.Net handles all steps in the transaction process — such as payment data collection, data submission, and response to the customer — while the customer remains in your store. Authorize.Net Direct Post (Deprecated) can be used with orders created from the Admin as well as from the store.
+{:.bs-callout-warning}
+**Deprecation Notice** <br/>
+Due to the Payment Service Directive [PSD2]({% link stores/compliance-payment-services-directive.md %}) and the continued evolution of many APIs, this payment integration is at risk of becoming outdated and no longer security compliant in the future. Additionally, Authorize.Net has deprecated the Authorize.Net Direct Post payment method. We are recommending that you disable it in your Magento configuration and transition to the Authorize.Net [Magento Marketplace extension](https://marketplace.magento.com/catalogsearch/result/?q=authorize.net){:target="_blank"}.<br/><br/>
+**This integration will be removed from the Magento 2.4.0 release and has been deprecated from versions of 2.3.**<br/><br/>
```

Comment:
not sure what is meant here.   deprecated from versions of 2.3.  is it all 2.3 versions?

should it be "has been deprecated in 2.3.x versions."

Reply:
This was Misha's language in his DevBlog.  I can go either way, but decided to stick to his since we are refering to his article for details.